src/core_ocean/shared/mpas_ocn_tendency.F

```diff
@@ -326,6 +336,31 @@ subroutine ocn_tend_vel(tendPool, statePool, forcingPool, diagnosticsPool, meshP
       !
       call ocn_vel_vadv_tend(meshPool, normalVelocity, layerThicknessEdge, vertAleTransportTop, tend_normalVelocity, err)
 
+      !
+      ! velocity tendency: tidal potential (if needed) 
+      !   For RK4, subtract the tidal potential from the zMid array and store in a work array, 
+      !   Then point zMid to work array so the tidal potential terms are included inside the grad computed in ocn_vel_pressure_grad_tend
+      !   zMid is pointed back to the typical value afterward, so the work array is not used in place of zMid elsewhere.
+      !
+      if (config_use_tidal_potential_forcing) then
+        call ocn_compute_tidal_potential_forcing(meshPool, forcingPool, diagnosticsPool, err)
+      end if
+
+      if (config_use_tidal_potential_forcing .and. config_time_integrator == 'RK4') then
+        call mpas_pool_get_array(forcingPool, 'tidalPotentialZMid', tidalPotentialZMid)
+        call mpas_pool_get_array(forcingPool, 'tidalPotentialEta', tidalPotentialEta)
+        call mpas_pool_get_array(meshPool, 'maxLevelCell', maxLevelCell)
+
+        do iCell = 1, nCells
+          do k = 1, maxLevelCell(iCell)
+            tidalPotentialZMid(k,iCell) = zMid(k,iCell) - tidalPotentialEta(iCell) &
```

Comment:
@sbrus89, potential issue may be here with #366 including tidal forcing.  `zMid` is a strong negative with the spurious datum shift we are employing in #366 until the vertical datum issue is resolved.